doc/sources/installation/installation-osx.rst

```diff
@@ -85,7 +128,7 @@ Alternatively you can install Kivy using the following steps:
 
         $ brew install sdl2 sdl2_image sdl2_ttf sdl2_mixer gstreamer
 
-    2. Install cython 0.21.2 and kivy using pip::
+    2. Install cython 0.23 and kivy using pip::
 
         $ pip install -I Cython==0.21.2
-        $ USE_OSX_FRAMEWORKS=0 pip install git+https://github.com/kivy/kivy.git@1.9.0
+        $ USE_OSX_FRAMEWORKS=0 pip install git+https://github.com/kivy/kivy.git@1.9.1
```

Comment:
This can be simplified to:
`$ USE_OSX_FRAMEWORKS=0 pip install kivy`

or with https://github.com/kivy/kivy/pull/3685 even just:
`$ pip install kivy`